python_backend/schemas/survey.py

```
@@ -0,0 +1,22 @@
+
+    class Config:
+        # 예시 데이터를 Swagger UI에 표시
+        json_schema_extra = {
+            "example": {
+                "company_name": "ChatToner Corp",
+                "team_size": 20,
+                "communication_style": "friendly",
+                "main_channel": "Slack",
+                "main_target": ["내부 동료", "타팀/타부서"]
+            }
+        }
```

Ruff (0.13.1)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)